[Notifier] Add Bandwidth bridge (#17483)
symfony/symfony-docs: notifier.rst

    The ContactEveryone and SMSFactor integrations were introduced in Symfony 6.2.

.. versionadded:: 6.3

>     The Bandwidth, Plivo, RingCentral, and Termii integrations were introduced in Symfony 6.3.

Reviewer: ```suggestion
    The Bandwidth integration was introduced in Symfony 6.3.
```

We need to add the other ones later
